<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StanStolikow</class>
 <widget class="QDialog" name="StanStolikow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_stolik1_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_stolik1_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>122</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_czyWolny2">
      <property name="text">
       <string>Brak</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_zwolnijStolik2">
      <property name="text">
       <string>Zwolnij</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_stolik1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>122</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_czyWolny1">
      <property name="text">
       <string>Brak</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ZwolnijStolik1">
      <property name="text">
       <string>Zwolnij</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>122</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_czyWolny3">
      <property name="text">
       <string>Brak</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_zwolnijStolik3">
      <property name="text">
       <string>Zwolnij</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>590</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>cofnij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cofnij</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Kwota</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_kwota1_2">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Kwota</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_kwota1_3">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Kwota</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_kwota1">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
